### ISSUE TITLE
[convert] Create a winesapos-convert script

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#724]
The `winesapos-convert.sh` script should convert any Arch Linux based operating system to provide a similar experience to install winesapOS. This has the advantage of allowing users to use an installer (such as `archinstall` or a Manjaro live ISO) while being able to convert to a winesapOS-like installation.

Goals:
- Manage Flatpaks.
- Manage Pacman package repositories.
- Manage Pacman packages.
- Manage desktop shortcuts.
- System settings (some but not all).
- Support containers (such as distrobox and toolbox managed containers).

Non-goals:
- Manage upgrades.
    - Instead, refer to this new project: https://github.com/LukeShortCloud/winesapOS/issues/711
- Manage Linux kernels.
- Manage storage devices.

Undecided:
- Support other operating systems besides Arch Linux, EndeavourOS, and Manjaro (ex., Fedora).

**Comment by LukeShortCloud** [comment]
Although most of our scripts are written in Zsh, this should be written in Bash for better cross-platform compatibility. Long-term, we want to convert our scripts to Bash anyways to lower the barrier to entry for new contributors.

**Comment by LukeShortCloud** [comment]
This conversion script would replicate as close as possible our performance release image type.

**Comment by LukeShortCloud** [comment]
Alternatively, we can go back to our original use of idempotent Ansible roles and a playbook from Mac Linux Gaming Stick 1.X.Y.

There are two reasons we switched to shell scripts instead: (1) the future of the Ansible project was unknown as a major restructure happened upstream, (2) it was limiting in customization of the build, and (3) Ansible required a handful of dependencies to be installed. Fast-forward to today, (1) Ansible is stable again as the dust as settled from the restructure, (2) most if not all of our users do not care about customizing the build (there are still some minor things we came expose such as allowing additional packages to be installed, etc. but primarily they can use additional roles), and (3) we can easily provide a wrapper script or at least instructions on how to install those dependencies.

https://github.com/LukeShortCloud/winesapOS/tree/1.2.1
